IT-Wizards-API/src/main/java/org/LaunchCode/IT_Wizards_API/controllers/WishlistController.java

```diff
@@ -0,0 +1,33 @@
+package org.LaunchCode.IT_Wizards_API.controllers;
+
+import org.LaunchCode.IT_Wizards_API.exceptions.WishlistNotFoundException;
+import org.LaunchCode.IT_Wizards_API.models.Wishlist;
+import org.LaunchCode.IT_Wizards_API.repository.WishlistRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/wishlist")
+public class WishlistController {
+
+    @Autowired
+    private WishlistRepository wishlistRepository;
+
+    @PostMapping()
+    Wishlist newWishlist(@RequestBody Wishlist newWishlist) {
+        return wishlistRepository.save(newWishlist);
+    }
+
+    @GetMapping()
+    List<Wishlist> getAllWishlists() {
+        return wishlistRepository.findAll();
+    }
+
```

Comment:
Wondering if the GetMapping should use the User class to only return the wishlists for the specific user?

Reply:
I was thinking of this as well, along with the cart controller. We can work towards making restrictions with website visitors to only be allowed to add to cart/wishlist if they are logged in.